pycozmo/audio.py

```diff
@@ -0,0 +1,147 @@
+import struct
+import time
+import wave
+from datetime import datetime, timedelta
+from threading import Thread, Lock
+
+from .conn import ClientConnection
+from .protocol_encoder import OutputAudio, Keyframe
+
+MIN_WAIT = 0.033
+
+class AudioManager():
+    """
+    This class takes care of reading audio files and generating the OutputAudio messages sent to
+    cozmo.
+
+    The play() method can be used to play an audio file or list of OutputAudio messages.
+
+    Args:
+        conn (ClientConnection): client managing the communication with the robot
+    """
```

Comment:
It will be great if you could modify `examples/audio.py` to use the new `Client.play_audio()`.